src/etl/sds/executor/input/tests/test_input_trigger.py

```diff
@@ -0,0 +1,575 @@
+import json
+import os
+import time
+import uuid
+from collections import deque
+from functools import partial
+from types import FunctionType
+from unittest import mock
+
+import boto3
+import pytest
+from botocore.exceptions import ClientError
+from etl_utils.constants import CHANGELOG_NUMBER, ETL_STATE_LOCK, WorkerKey
+from etl_utils.io import pkl_dumps_lz4
+from etl_utils.io.test.io_utils import pkl_loads_lz4
+from etl_utils.trigger.model import _create_timestamp
+from etl_utils.trigger.operations import StateFileNotEmpty
+from moto import mock_aws
+from mypy_boto3_s3 import S3Client
+from mypy_boto3_stepfunctions import SFNClient
+
+from etl.sds.executor.input.steps import _start_execution
+from test_helpers.terraform import read_terraform_output
+
+MOCKED_INPUT_TRIGGER_ENVIRONMENT = {
```

Comment:
Might be worth moving all these constants to a separate file so that the tests are the main star of this file?